src/shared/util.js

```diff
@@ -1219,16 +1219,111 @@ function MessageHandler(sourceName, targetName, comObj) {
   this.targetName = targetName;
   this.comObj = comObj;
   this.callbackIndex = 1;
+  this.streamId = 1;
```

Comment:
I just noticed that there's a couple of fairly long lines in the patch already (for example [this one](https://github.com/mozilla/pdf.js/pull/8430/commits/9444da8061e4bfc2c7a827f34752000bbc089fd5#diff-8d2b578d58d4d8c8a2975020fbbc84d9R1263)), and it's possible that using `streamIndex` instead of `streamId` everywhere could perhaps unnecessarily go over the maximum limit of 80 characters per line.

However, it seems that we already use `callbackId` in most places in the existing code, hence I'm suggesting to keep `this.streamId` in this patch and just change the *two* existing occurrences of `this.callbackIndex` to `this.callbackId` instead.

Sorry about given confusing/conflicting advice here!